backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -204,10 +213,38 @@ export class ContributionResolver {
 
     // all possible cases not to be true are thrown in this function
     validateContribution(creations, amount, creationDateObj)
+
+    const contributionMessage = ContributionMessage.create()
+    contributionMessage.contributionId = contributionId
+    contributionMessage.createdAt = contributionToUpdate.updatedAt
+      ? contributionToUpdate.updatedAt
+      : contributionToUpdate.createdAt
+    let changeMessage = ''
+    if (contributionToUpdate.memo !== memo) {
+      // TODO: Add message to changeMessage
+      changeMessage += 'Alter Memo: ' + contributionToUpdate.memo + '\n'
```

Comment:
I suggest no text. Just display the whole contribution as it was before the update.

Reply:
Withdrew the textes